setup.py

```diff
@@ -24,6 +24,14 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/prjxray",
     packages=['prjxray'],
+    install_requires=[
+        'fasm',
+        'intervaltree',
+        'numpy',
+        'pyjson5',
```

Comment:
Where is `pyjson5` used?

Reply:
And what happened to `sdf-timing`?  Is it not required?

Reply:
> Where is pyjson5 used?

https://github.com/SymbiFlow/prjxray/blob/master/prjxray/lib.py
https://github.com/SymbiFlow/prjxray/blob/master/prjxray/node_lookup.py

> And what happened to sdf-timing

This seems not to be used by any of the scripts belonging to the python package

Reply:
> > Where is pyjson5 used?
> 
> https://github.com/SymbiFlow/prjxray/blob/master/prjxray/lib.py
> https://github.com/SymbiFlow/prjxray/blob/master/prjxray/node_lookup.py

Both of these probably shouldn't be in the prjxray library, as they operate on data that is only present in prjxray fuzzer directories.   Let's leave pyjson5 in for now, but add a README to isolate those files and eventually remove this dependency.